tests/QueueReceiver.IntegrationTests/WorkerServiceTests.cs

```diff
@@ -0,0 +1,100 @@
+using Microsoft.Azure.ServiceBus;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Newtonsoft.Json.Linq;
+using QueueReceiver.Core.Interfaces;
+using QueueReceiver.Core.Models;
+using QueueReceiver.Core.Services;
+using QueueReceiver.Worker;
+using System;
+using System.Net;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace QueueReceiver.IntegrationTests
+{
+    [TestClass]
+    public class WorkerServiceTests
+    {
+        private const int MillisecondsDelay = 1000;
+
+        private IConfiguration Configuration { get;  }
+
+        public WorkerServiceTests()
+        {
+            WebRequest.DefaultWebProxy = new WebProxy("http://www-proxy.statoil.no:80");
```

Comment:
Should this be in Configuration ?

Reply:
Probably, we have the same in Worker.Program. I've not 100% figure out how to set it properly yet. There is a TODO :)